### PR TITLE
Update examples from ballerina-1.2.x branch

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import ballerina/http;
+import ballerina/knative;
+import ballerina/log;
+
+//Add the `@knative:Service` to a Ballerina service to generate a Knative Service artifact and push the Docker image to Docker Hub.
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal:5]:
```suggestion
//Adds the `@knative:Service` to a Ballerina service to generate a Knative Service artifact and push the Docker image to Docker Hub.
```

File: kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+@knative:Service {
+    //Enable pushing the Docker image.
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal:7]:
```suggestion
    //Enables pushing the Docker image.
```

File: kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    push: true,
+    //Set the name of the Docker image.
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal:9]:
```suggestion
    //Sets the name of the Docker image.
```

File: kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    name: "hello-world-knative",
+    //Sets the username credential to push the Docker image using the `DOCKER_USERNAME` environment  variable.
+    username: "$env{DOCKER_USERNAME}",
+    //Sets the password credential to push the Docker image using the `DOCKER_PASSWORD` environment  variable.
+    password: "$env{DOCKER_PASSWORD}",
+    //Setting the registry URL.
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/knative-deployment/knative_deployment.bal:15]:
```suggestion
    //Sets the registry URL.
```

File: kubernetes-extension-examples/examples/knative-deployment/knative_deployment.description [hunk 5]
```diff
@@ -0,0 +1,3 @@
+//Ballerina supports generating Knative Serivces artifacts based on annotations.
+//Artifacts will be generated by adding the `@knative:Service` annotation to the HTTP service.<br/><br/>
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/knative-deployment/knative_deployment.description:2]:
```suggestion
//The artifacts will be generated by adding the `@knative:Service` annotation to the HTTP service.<br/><br/>
```

File: kubernetes-extension-examples/examples/openshift-deployment/.gitignore [hunk 6]
```diff
@@ -1 +1,2 @@
-kubernetes/
\ No newline at end of file
+kubernetes/
+docker/
```
Comment (praneesha) [inline kubernetes-extension-examples/examples/openshift-deployment/.gitignore:2]: Might need to keep a new line at the end of the file.

Comment (hemikak) [inline kubernetes-extension-examples/examples/openshift-deployment/.gitignore:2]:
```suggestion
docker/

```

File: kubernetes-extension-examples/examples/knative-deployment/.gitignore [hunk 7]
```diff
@@ -0,0 +1,2 @@
+kubernetes/
+docker/
```
Comment (hemikak) [inline kubernetes-extension-examples/examples/knative-deployment/.gitignore:2]:
```suggestion
docker/

```

**Pull request merged**